examples/pytorch/seal/main.py

```diff
@@ -0,0 +1,175 @@
+import dgl
+from utils import parse_arguments
+from tqdm import tqdm
+from dgl import NID, EID
+from torch.nn import BCEWithLogitsLoss
+from utils import load_ogb_dataset, evaluate_hits
+import torch
+from sampler import SEALData, SEALDataLoader
+from model import GCN, DGCNN
+import numpy as np
+from logger import LightLogging
+import time
+
+
+def train(model, dataloader, loss_fn, optimizer, device, num_graphs=32, total_graphs=None):
```

Comment:
Train and eval logic are similar with the official implementation. Please add a comment on that, similar as: https://github.com/dmlc/dgl/blob/master/examples/pytorch/pointcloud/pointnet/pointnet2.py#L10